aries-site/src/examples/components/tag/CreateTagSimple.js

```diff
@@ -0,0 +1,152 @@
+import React, { useContext, useState } from 'react';
+import {
+  Box,
+  Button,
+  Grid,
+  Form,
+  FormField,
+  Select,
+  Tag,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { ThemeContext } from 'styled-components';
+import { simpleTags } from './data';
+import {
+  alphabetize,
+  AppContainer,
+  getRegExp,
+  TagPageHeader,
+  prefix,
+  TagResults,
+  updateCreateOption,
+} from './CreateTag';
+
+const defaultValueOptions = [];
+const defaultValue = '';
+const defaultTags = [];
+
+simpleTags.forEach(datum => {
+  defaultValueOptions.push(datum);
+});
+
+export const CreateTagSimple = () => {
+  const size = useContext(ResponsiveContext);
+  const theme = useContext(ThemeContext);
+
+  // value
+  const [valueOptions, setValueOptions] = useState(
+    alphabetize(defaultValueOptions),
+  );
+  const [value, setValue] = useState(defaultValue);
+  const [valueSearch, setValueSearch] = useState('');
+
+  // tags
+  const [currentTags, setCurrentTags] = useState(defaultTags);
+
+  return (
+    <AppContainer>
+      <TagPageHeader
+        description={`Tags are values that can be assigned to 
+      resources.`}
+      />
+      <Form
+        messages={{ required: 'This is a required field.' }}
+        onSubmit={() => {
+          const exists = currentTags.filter(t => t === value);
+          if (!exists.length) setCurrentTags([...currentTags, value]);
+          setValue(defaultValue);
+        }}
+      >
+        <Grid
+          columns={
+            size !== 'small'
+              ? [['auto', 'medium'], ['auto', 'medium'], 'auto']
```

Comment:
This grid columns definition seems to have 3 cols when we only need 2.
```suggestion
              ? [['auto', 'medium'], 'auto']
```

![Screen Shot 2022-01-10 at 2 29 41 PM](https://user-images.githubusercontent.com/1756948/148842371-a9555b85-a064-4d2d-b0e8-f4d142ff5d85.png)